kafka/util.py

```diff
@@ -67,6 +67,7 @@ class Dict(dict):
     pass
 
 
+# TODO what is the purpose of this? It has been unused since it was originally added
```

Comment:
I was thinking about using this to help prevent circular references when calling instance methods with `atexit` . Currently we only do that in the producer, and we use weakproxy directly. If we ever needed to do a similar cleanup for other classes then this method might be helpful. But I don't mind if you delete it.